app/models/fs/message_draft.rb

```diff
@@ -31,7 +31,12 @@ def self.create_and_validate_with_fs_form(form_files: [], author:, fs_client: Fs
 
     form_files.each do |form_file|
       form_content = form_file.read.force_encoding("UTF-8")
-      form_information = fs_client.api.parse_form(form_content)
+      begin
+        form_information = fs_client.api.parse_form(form_content)
```

Comment:
Toto by som strcil dovnutra, ak je ocakavane, ze nastane validacna chyba, tak by sme to nemali riesit exception. rep. urcite nechytajme vsetky vynimky takto ale len tie co mozu naozaj nastat lebo to bude nedebugovtelne.